compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -2455,7 +2455,7 @@ declare_lint! {
     ///
     /// ### Example
     ///
-    /// ```rust
+    /// ```rust,ignore (fails on non-x86_64)
     /// #[cfg(target_arch="x86_64")]
```

Comment:
```suggestion
    /// #![cfg(target_arch="x86_64")]
```

Reply:
That won't work. The tests are checking that the lint actually triggers in the example, but it only triggers on x86_64.